creator-node/src/routes/files.js

```diff
@@ -538,6 +538,53 @@ module.exports = function (app) {
     return successResponse(cidExistanceMap)
   }))
 
+  /**
+   * Serves information on existence of given image cids
+   * @param req
+   * @param req.body
+   * @param {string[]} req.body.cids the cids to check existence for, these cids should be directories containing original.jpg
+   * @dev This route can have a large number of CIDs as input, therefore we use a POST request.
+   */
+  app.post('/batch_image_cids_exist', handleResponse(async (req, res) => {
+    const { cids } = req.body
+
+    if (cids && cids.length > BATCH_CID_ROUTE_LIMIT) {
+      return errorResponseBadRequest(`Too many CIDs passed in, limit is ${BATCH_CID_ROUTE_LIMIT}`)
+    }
+
+    const queryResults = (await models.File.findAll({
```

Comment:
did we decide to not add a `type != image` for the non-image batch CID route?

Reply:
Yes, because we still want to support the existence of directory check.

Reply:
right - sounds good, can you add comment to that effect 🙂

Reply:
```
   * @param {string[]} req.body.cids the cids to check existence for, these cids can also be directories

```

This comment should explain that already, if you think that is sufficient, I think this is good to merge now too.